ddtrace/internal/serverless/mini_agent.py

```diff
@@ -1,29 +1,50 @@
 import os
 from subprocess import Popen
+import sys
 
-from . import in_gcp_function
 from ..compat import PYTHON_VERSION_INFO
 from ..logger import get_logger
+from ..serverless import in_azure_function_consumption_plan
+from ..serverless import in_gcp_function
 
 
 log = get_logger(__name__)
 
 
 def maybe_start_serverless_mini_agent():
-    if not in_gcp_function():
+    if not (in_gcp_function() or in_azure_function_consumption_plan()):
         return
 
-    try:
-        python_folder_name = "python%d.%d" % PYTHON_VERSION_INFO[:2]
-
-        rust_binary_path_default = (
-            "/layers/google.python.pip/pip/lib/%s/site-packages/"
-            "datadog-serverless-agent-linux-amd64/datadog-serverless-trace-mini-agent"
-        ) % python_folder_name
+    if sys.platform != "win32" and sys.platform != "linux":
```

Comment:
This will miss windows 64 and something [unsupported](https://docs.python.org/3/library/sys.html#sys.platform) platforms. Should we use `sys.platform == "linux"` instead?

Reply:
I'm a bit confused by this, `maybe_start_serverless_mini_agent` should only be called in google cloud functions which only supports linux, & azure functions which only supports linux + windows (though python support is limited to linux).

win32 should indicate 64bit windows though the log line in this if statement is a bit redundant since this functions shouldn't be called outside of windows + linux anyways